router/transformer/transformer.go

```diff
@@ -200,22 +201,19 @@ func (trans *HandleT) ProxyRequest(ctx context.Context, responseData integration
 		var requestError error
 		//start
 		rdl_time := time.Now()
-		respData, respCode, requestError = trans.makeHTTPRequest(ctx, url, payload)
-		if requestError != nil {
-			stats.NewTaggedStat("transformer_proxy.request_latency", stats.TimerType, stats.Tags{"requestSuccess": "false"}).SendTiming(time.Since(rdl_time))
-			stats.NewTaggedStat("transformer_proxy.request_result", stats.CountType, stats.Tags{"requestSuccess": "false"}).Increment()
-		} else {
-			stats.NewTaggedStat("transformer_proxy.request_latency", stats.TimerType, stats.Tags{"requestSuccess": "true"}).SendTiming(time.Since(rdl_time))
-			stats.NewTaggedStat("transformer_proxy.request_result", stats.CountType, stats.Tags{"requestSuccess": "true"}).Increment()
-		}
+		respData, respCode, requestError = trans.makeHTTPRequest(ctx, url, payload, destName)
+		stats.NewTaggedStat("transformer_proxy.delivery_response", stats.CountType, stats.Tags{"destination": destName}).Increment()
+		reqSuccessStr := strconv.FormatBool(requestError != nil)
```

Comment:
This check should be inverted to match the `requestSuccess` definition.

Reply:
Yeah. Changed it